tac/project-updates/2024/2024-Q4-Hyperledger-Iroha.md

```diff
@@ -0,0 +1,56 @@
+
+Additionally, the Iroha 2 team has finished developing the Iroha 2 [Blockchain Explorer](https://github.com/soramitsu/iroha2-block-explorer-web). The MVP is now in great shape and ready for use.
+
+We also successfully moved Iroha 2’s codebase to its [own repository](https://github.com/hyperledger-iroha) as part of the transition from the Hyperledger Foundation to the Linux Foundation’s Decentralized Trust initiative.
+
+The Iroha 2 community is active and growing, with lots of interest from potential users and contributors. Questions are being answered regularly, five issues were assigned to the maintainers among community members, and  the PRs for the ones have been merged since the last update. Project updates are being sent every two weeks to keep everyone connected and informed.
+
+# Questions/Issues for the TAC
+
+None at this time.
+
+# Releases
+
+[2.0.0-pre-rc.22.0](https://github.com/hyperledger-iroha/iroha/releases/tag/v2.0.0-pre-rc.22.0)  
+[2.0.0-pre-rc.22.1](https://github.com/hyperledger-iroha/iroha/releases/tag/v2.0.0-pre-rc.22.1)  
+[2.0.0-pre-rc.22.2](https://github.com/hyperledger-iroha/iroha/releases/tag/v2.0.0-pre-rc.22.2)
+
+Please note: that we are still in the process of finalizing and publishing the MVP PR.  
+Summary of Activities in the Last Quarter related to Community management
+
+# Overall Activity in the Past Quarter
+
+Iroha 2 has been actively engaging with the community through various channels, including Telegram and Discord, where we assist with technical questions, gather feedback, and provide technical guidance. 
+
+The team also hosts a weekly open [community call](https://meet.jit.si/Iroha_biweekly), where we share the latest updates and address questions from community members. Additionally, Iroha 2 sends out a regular newsletter to the [Hyperledger mailing list](https://lists.lfdecentralizedtrust.org/g/iroha), keeping everyone informed on the latest development updates and releases. All while ensuring that communication follows our [code of conduct](https://lf-decentralized-trust.github.io/governance/governing-documents/code-of-conduct.html).
+
+# Current Plans
+
+* Release the MVP and ensure all SDKs are functioning properly.  
+* Iroha 2 has a well-maintained development roadmap for the post-MVP phase. High-priority items include:  
+  * Network scalability  
+  * Network performance and optimization  
+  * Further development and enhancement of user-facing technical documentation
+
+# Maintainer Diversity
+
+Since the last update, the composition of the Iroha 2 maintainers has changed significantly. For the most up-to-date information, please refer to the [Maintainers list](https://github.com/hyperledger-iroha/iroha/blob/main/MAINTAINERS.md).
```

Comment:
It would be nice to have more details on how/what has changed? Did this change the diversity of the maintainers?

Reply:
Added details about specific participants come and go